pkg/security/tests/mount_test.go

```diff
@@ -367,7 +367,7 @@ func TestMountSnapshot(t *testing.T) {
 		assert.Equal(t, dev, mount.Device, "snapshot and model device mismatch")
 		assert.Equal(t, mntInfo.FSType, mount.FSType, "snapshot and model fstype mismatch")
 		assert.Equal(t, mntInfo.Root, mount.RootStr, "snapshot and model root mismatch")
-		mountPointPath, err := mountResolver.ResolveMountPath(mount.MountID, dev, pid, "")
+		mountPointPath, _, _, err := mountResolver.ResolveMountPath(mount.MountID, dev, pid, "")
```

Comment:
could we add some checks on new returned values?

Reply:
Done